rasa_core_sdk/forms.py

```diff
@@ -443,7 +472,11 @@ def _validate_if_required(self, dispatcher, tracker, domain):
             - the form is called after `action_listen`
             - form validation was not cancelled
         """
-        if tracker.latest_action_name == "action_listen" and tracker.active_form.get(
+        if tracker.latest_action_name == "form" and tracker.active_form.get(
+            "validate", True
+        ):
+            return self._validate_prefilled_slots(dispatcher, tracker, domain)
+        elif tracker.latest_action_name == "action_listen" and tracker.active_form.get(
```

Comment:
this if looks awful, is it black formatting?

Reply:
yup

Reply:
can we reformat it? and put `and ...` on the new line?

Reply:
I tried 
```
elif (
      tracker.latest_action_name == "action_listen"
      and tracker.active_form.get("validate", True)
):
``` 
and no dice, black keeps changing it back

Reply:
^ above is not good for black @tabergma

Reply:
If black formats the line this way, we cannot do much about it. Sorry.